functions/controllers/user/imageUpload.js

```diff
@@ -0,0 +1,56 @@
+const Busboy = require('busboy')
+const path = require('path')
+const os = require('os');
+const fs = require('fs')
```

Comment:
Your import should be alphabetical

Reply:
see article https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d